spectrum_utils/proforma_simple.ebnf

```diff
@@ -0,0 +1,61 @@
+%import common.DIGIT
```

Comment:
Sounds good! the gain it not large enough for it to be worth it ... since lark can run in Pypy there should be a way to numba-optimize or use this pluign https://github.com/lark-parser/lark_cython ... I will look into it at a later time (will roll back the simple grammar implementation)

Reply:
lark-Cython seems interesting to check out indeed!